monkey/infection_monkey/network/postgresql_fingerprint.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+import psycopg2
+
+from infection_monkey.network.HostFinger import HostFinger
+
+LOG = logging.getLogger(__name__)
+
+
+class PostgreSQLFinger(HostFinger):
+    """
+    Fingerprints PostgreSQL databases, only on port 5432
+    """
+    # Class related consts
+    _SCANNED_SERVICE = 'PostgreSQL'
+    POSTGRESQL_DEFAULT_PORT = 5432
```

Comment:
Wheres the addition of this port to check?

Reply:
Addition of this port to check?
It's being used [here](https://github.com/guardicore/monkey/pull/892/files#diff-b9c3716cd22e63ad02327df4ae2ec50e10b782f9507e7f55e9395dd2784d44f6R23), if that's what you're asking?

Reply:
It's worth adding to tcp_target_ports in `config.py` so it'll be generically checked as well. Then if that port is off you can save the expensive check using pg

Reply:
Added

Reply:
So if the port of postgress is not 5432 we will not fingerprint it?

Reply:
How will we discover that port? It won't be 22...

Reply:
We could check all ports under config -> internal -> network -> TCP target ports. In that case, if user wants to scan postgres with custom port, he can still do that. Not a requirement, just an idea, which should probably be applied to all fingerprinters.